Vuln db search failure (#377)
rapid7/insightconnect-plugins: rapid7_vulndb/help.md

## Setup

_This plugin does not contain a connection._

## Technical Details

### Actions

#### Get Content

> This action is used to fetch content record for vulnerability or module.

Reviewer: ```suggestion
This action is used to fetch content results for a vulnerability or module.
```

Consistency with the output type

_This plugin does not contain any triggers._

### Custom Output Types

#### content

|Name|Type|Required|Description|
|----|----|--------|-----------|
|Alternative identifiers|string|False|List of alternative identifiers of vulnerability|
|architectures|string|False|List of applicable architectures for the module|
|Authors|string|False|List of module authors|
|Content type|string|False|Type of returned content, module or vulnerability|
|Description|string|False|Brief summary of the returend content|
|Published_at|string|False|Published date of vulnerability|
|Rank|int|False|Rank of module|
|References|string|False|List of references|
|Reliability|string|False|Reliability of module|
|Severity|string|False|Severity of vulnerability|
|Solutions|string|False|List of possible solutions for vulnerability|
|Title|string|False|Title of Vulnerability|

#### search_result

|Name|Type|Required|Description|
|----|----|--------|-----------|
|Link|string|False|Link to vulnerability|
|Published_at|string|False|Published date of vulnerability|
|Title|string|False|Title of Vulnerability|

## Troubleshooting

_This plugin does not contain any troubleshooting information._

# Version History

> * 2.0.0 - New version based on API, not backward compatible

Reviewer: ```suggestion
* 2.0.0 - Utilize VulnDB API
```

## References

> 

Reviewer 2: Keep this but rename to `Rapid7 Vulnerability & Exploit Database`

> * [Rapid7 Vulnerability & Exploit Database](https://vdb.rapid7.com/swagger_doc)

Reviewer 2: ```suggestion
* [Rapid7 Vulnerability & Exploit Database API Specification](https://vdb.rapid7.com/swagger_doc)
```
